webapp/channels/src/actions/storage.ts

```diff
@@ -10,10 +10,13 @@ export function setGlobalItem(name: string, value: any) {
     };
 }
 
-export function removeGlobalItem(name: string) {
-    return {
-        type: StorageTypes.REMOVE_GLOBAL_ITEM,
-        data: {name},
+export function removeGlobalItem(name: string): NewActionFunc {
+    return (dispatch) => {
+        dispatch({
+            type: StorageTypes.REMOVE_GLOBAL_ITEM,
+            data: {name},
+        });
+        return {data: true};
     };
 }
```

Comment:
This needs to be unreverted since it refers to a now nonexistent type
```suggestion
export function removeGlobalItem(name: string) {
    return {
        type: StorageTypes.REMOVE_GLOBAL_ITEM,
        data: {name},
    };
}
```